src/platform/datapath_winuser.c

```diff
@@ -357,6 +357,11 @@ typedef struct CXPLAT_DATAPATH_PROC {
     //
     uint16_t Index;
 
+    //
+    // Flag indicates that the worker thread is currently actively processing.
+    //
+    uint16_t WorkerActive : 1;
```

Comment:
Is there a reason to make this a unit16_t and a bitflag? Unless we plan on making operations with this interlocked.

Reply:
Only a bit is needed. And there was space for a `uint16_t`.